src/App.js

```diff
@@ -3,9 +3,9 @@ import {
   Routes,
   Route } from "react-router-dom"
 
-import Home from "./pages/Home";
-import Books from "./pages/Books";
-import Authors from "./pages/Authors";
+import Home from "./pages/Home/Home";
+import Books from "./pages/Books/Books";
+import Authors from "./pages/Authors/Authors";
```

Comment:
I created a new directory for each one these pages, the reason was to put any component that seemed directly related to these pages near to the parent component. If you have a base `/src/components` directory, generally you only want to put components in there that can be used by any page.